app/Http/Controllers/Assets/AssetsController.php

```diff
@@ -131,9 +131,6 @@ public function store(ImageUploadRequest $request)
             $asset->order_number            = $request->input('order_number');
             $asset->notes                   = $request->input('notes');
             $asset->user_id                 = Auth::id();
-            $asset->archived                = '0';
-            $asset->physical                = '1';
-            $asset->depreciate              = '0';
```

Comment:
It's possible we put these defaults in here so that MySQL doesn't throw up on them if they're null and the db field is not nullable.

Reply:
> `archive` and `depreciate` are both defaulted at the db level to what we were setting them to, so we shouldn't notice any difference there. `physical` is not defaulted at the db level, but it is nullable, and I couldn't find any evidence of it being used anywhere.

I think we're good and I could never get it to error.

Reply:
We've been bitten by that before though. If you pass it and it's null, without mutators (like we did for qty), it can cause MySQL to crash out.

Reply:
<img width="717" alt="image" src="https://github.com/snipe/snipe-it/assets/7305753/8307266e-a943-4648-bcad-1e183fdd4d7c">

Reply:
<img width="717" alt="image" src="https://github.com/snipe/snipe-it/assets/7305753/8307266e-a943-4648-bcad-1e183fdd4d7c">

Seems fine, writing a real test for it now.

Reply:
Tests added.